airbyte-integrations/connectors/source-twitter/source_twitter/schemas/authors.json

```diff
@@ -0,0 +1,91 @@
+{
+  "$schema" : "http://json-schema.org/draft-07/schema#",
+  "type" : "object",
+  "properties" : {
+    "id" : {
```

Comment:
Make sure the schema is following the ref: https://docs.airbyte.com/connector-development/schema-reference